test/pytest/receiver/plugins/test_asapo_receiver.py

```diff
@@ -18,22 +18,20 @@
 def config():
     config = dict(
         endpoint="asapo-services:8400",
-        beamline="p00",
-        beamtime="auto",
+        beamtime="p00",
         token="abcdefg1234=",
         data_source='test001',
         n_threads=1,
-        ingest_mode="INGEST_MODE_TRANSFER_METADATA_ONLY",
         file_regex=".*/(?P<detector>.*)/(?P<scan_id>.*)_scan[0-9]*-(?P<file_idx_in_scan>.*).tif",
-        ignore_regex=".*/.*.metadata$",
         user_config_path="Path"
     )
     return config
 
 
 @pytest.fixture
 def worker(config):
-    worker = AsapoWorker(config)
+    config = {k: v for k, v in config.items() if k != "user_config_path"}
```

Comment:
Maybe easier to read
```suggestion
config = config.copy()
del config["user_config_path"]
```

Reply:
Changed.